### PR TITLE
Add events api. Update eventList and eventInfo components.

### PR DESCRIPTION
**Pull request by AlexOrd**

### CODE REVIEW

File: src/actions/calendarDataActions.js [hunk 1]
```diff
@@ -38,7 +38,8 @@ export function loadEvents() {
 export function getCalendarList() {
   return function(dispatch) {
     eventsAPI.calendarList().then(response => {
-      const list = response.result.items;
+      // filter primary calendar
```
Comment (MariiaPolova) [inline src/actions/calendarDataActions.js:41]: what about other side calendars like contacts? is there any?

File: src/components/main/HomePage.js [hunk 2]
```diff
@@ -16,32 +17,34 @@ import { withStyles } from '@material-ui/core/styles';
...
     return (
-      <Paper className={classes.root} elevation={1}>
-        <InfoContainer calendar={calendarData.calendar}/>
-        <div>
+      <Paper className={classes.root} elevation={1} >
+        <InfoContainer calendar={calendar} />
+        <div className={classes.menu}>
           <ClockContainer />
-          <ListContainer />
+          <ListContainer events={events} />
```
Comment (MariiaPolova) [inline src/components/main/HomePage.js]: you dispatched it before to redux, why don't use it from there?

File: src/utils/texts.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+export function emailToName(email) {
```
Comment (MariiaPolova) [inline src/utils/texts.js:1]: maybe you can just list the email? some of them might be not orthodox. Keep that as a side note

File: src/components/EventInfo/EventPage.js [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    super(props);
+  }
+
+  render () {
+    const { events, calendar, classes } = this.props;
+    const selectedId = this.props.match.params.id;
+    const currentEvent = events.find(event => event.id == selectedId)
+    const eventStartTime = moment(currentEvent.start.dateTime).format('HH:MM');
+    const eventEndTime = moment(currentEvent.end.dateTime).format('HH:MM');
+    const personsWithCreator = [{ email: currentEvent.creator.email, responseStatus: 'accepted'}]
```
Comment (MariiaPolova) [inline src/components/EventInfo/EventPage.js]: I think you can also include here those whose acceptance is pending (a good point to discuss with people on meeting regarding that)

**Review by MariiaPolova — APPROVED** [review]

File: src/components/main/HomePage.js (hunk 2 above)
Comment (AlexOrd) [reply src/components/main/HomePage.js]: fixed.

**Pull request merged**